tests/selenium/webui-selenium-test.py

```diff
@@ -72,66 +72,128 @@ def setUp(self):
         self.wait_for_url_and_click('/restore/')
         self.wait_and_click(By.XPATH, '//a[contains(@href, "/dashboard/")]', By.XPATH, '//div[@id="modal-001"]//button[.="Close"]')
-        #self.assertRegexpMatches(self.close_alert_and_get_its_text(), r'^Oops, something went wrong, probably too many files.')
+
         self.close_alert_and_get_its_text()
         self.logout()
 
-
-
     def test_restore(self):
-
-        # LOGGING IN:
         self.login()
-
-        # CHANGING TO RESTORE TAB:
         self.wait_for_url_and_click('/restore/')
         self.wait_and_click(By.XPATH, '(//button[@data-id="client"])', By.XPATH, '//div[@id="modal-001"]//button[.="Close"]')
-        
-        # SELECTING CLIENT:
-        # Selects the correct client
+
+        # Select correct client
         self.wait_and_click(By.LINK_TEXT, self.client)
-        
-        # FILE-SELECTION:
+
         # Clicks on file and navigates through the tree
         # by using the arrow-keys.
         pathlist = self.restorefile.split('/')
         for i in pathlist[:-1]:
             self.wait_for_element(By.XPATH, '//a[contains(text(),"%s/")]' % i).send_keys(Keys.ARROW_RIGHT)
         self.wait_for_element(By.XPATH, '//a[contains(text(),"%s")]' % pathlist[-1]).click()
 
-        # CONFIRMATION:
-        # Clicks on 'submit'
+        # Submit restore
         self.wait_and_click(By.XPATH, '//input[@id="submit"]')
-        # Confirms alert that has text 'Are you sure ?'
+
+        # Confirms alert
         self.assertRegexpMatches(self.close_alert_and_get_its_text(), r'^Are you sure[\s\S]$')
-        # switch to dashboard to prevent that modals are open
+
+        # switch to dashboard to prevent that modals are open before logout
         self.wait_and_click(By.XPATH, '//a[contains(@href, "/dashboard/")]', By.XPATH, '//div[@id="modal-002"]//button[.="Close"]')
-        #self.assertRegexpMatches(self.close_alert_and_get_its_text(), r"^Oops, something went wrong, probably too many files.")
         self.close_alert_and_get_its_text()
 
-        # LOGOUT:
+        # Logout
+        self.logout()
+
+    def test_run_default_job(self):
+        self.login()
+        self.wait_and_click(By.ID, 'menu-topnavbar-job')
+        self.wait_and_click(By.LINK_TEXT, 'Run')
+        # Open the job list
+        self.wait_and_click(By.XPATH, '(//button[@data-id="job"])')
+        # Select the first job
+        self.wait_and_click(By.XPATH, '(//li[@data-original-index="1"])')
+        # Start it
+        self.wait_and_click(By.ID, 'submit')
+        self.wait_and_click(By.ID, 'menu-topnavbar-dashboard')
         self.logout()
 
+    def test_run_configured_job(self):
+
+        driver = self.driver
+        driver.get(self.base_url + '/')
+        self.login()
+        self.wait_and_click(By.ID, 'menu-topnavbar-job')
+        self.wait_and_click(By.LINK_TEXT, 'Run')
+
+        Select(driver.find_element_by_id('job')).select_by_visible_text('BackupCatalog')
```

Comment:
BackupCatalog only makes sense when backuping the server itself. Better use an normal backup job.